libraries/jsonball/data/resource_parameters.json

```diff
@@ -0,0 +1,34 @@
+[
+ ["resource_history_bytes",
+  {
+   "time_unit" : "blocks",
+   "budget_time" : {"days" : 30},
+   "budget" : "15000000000",
+   "half_life" : {"days" : 15},
+   "drain_time" : {"hours" : 1},
+   "p_min" : 100.0
+  }
+ ],
+
+ ["resource_new_accounts",
+  {
+   "time_unit" : "seconds",
+   "budget_time" : {"days" : 30},
+   "budget" : 1000000,
```

Comment:
This budget needs to be witness votable.

The voting is NYI, but the behavior of the curve needs to not be tied to a specific budget.